server.elm

```diff
@@ -655,30 +682,168 @@ div#modify-menu div.keyvalues > div.keyvalueadder:hover {
   opacity: 1;
 }
 div#modify-menu input {
-  padding: 4px;
+  padding: 6px;
+  width: 100%;
 }
 div#modify-menu #newTagName {
-  width: 50%;
-  font-size: 1em;  
+  font-size: 1.4em;  
+  background-color: rgb(0, 178, 179);
+  font-family: monospace;
+  padding: 4px;
+  flex: 1;
 }
 div#modify-menu .tagname-info {
-  color: #d4d4d4;
   display: inline-block;
-  width: 50%;
+  border-radius: 4px;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  background: transparent;
+  color: whitesmoke;
+  flex: 2;
 }
 div#modify-menu input[type=radio] {
   width: initial;
   font-size: 1em;
 }
+
+div.keyvalue > span > input {
+  border-radius: 4px;
+  margin: 2px;
+}
+
+div.tagname-summary {
+  display: flex;
+  background-color: rgb(0, 178, 179);
+  border-radius: 0.4em;
+  padding: 4px;
+}
+
 .inline-input {
   background: transparent;
   color: white;
   border: none;
 }
 
+/* dom selector css */
+div.dom-selector-style {
+  margin: 2%;
+  border-radius: 0.4em;
+  padding: 0.4rem;
+  background: rgba(0, 212, 159, 0.6);
```

Comment:
Same everywhere you define a color or background color. Use meaningful names for variables